src/ishikawa_backend.erl

```diff
@@ -125,16 +142,67 @@ handle_call({tcbcast, Message}, _From, #state{actor=Actor,
     %% Transmit to membership.
     [send(Message, Peer) || Peer <- Members],
 
-    {reply, ok, State#state{vv=VClock}};
-handle_call({tcbdeliver, _Message, _Timestamp}, _From, State) ->
-    %% TODO: Implement me.
-    {reply, ok, State};
+    %% Add members to the queue of not ack messages.
+    ToBeAckQueue1 = [ToBeAckQueue | [{Message, Members}]],
```

Comment:
I don't think this is doing what you want.  This is going to put the entire `ToBeAckQueue` list as the first element of `ToBeAckQueue1`.

Did you mean something like this?

`ToBeAckQueue1 = ToBeAckQueue ++ [{Message, Members}]`